### PR TITLE
Test more pixels to make it more thorough.

### PR DESCRIPTION
**Pull request by zhenyao**
@kenrussell per your suggestion.  please review.

### CODE REVIEW

File: sdk/tests/conformance2/reading/read-pixels-pack-parameters.html [hunk 1]
```diff
@@ -115,12 +115,22 @@
...
-    shouldBe("pixel", "expectedColor");
+    // Check the first and last pixels of each row.
+    // TODO(zmo): Revisit this if row_length < width cases are allowed.
+    for (var row = 0; row < height; ++row) {
+        var bytesPerRow = actualWidth * bytesPerPixel + padding;
+        // First pixel
+        var pos = skipSize + bytesPerRow * row;
+        for (var i = 0; i < bytesPerPixel; ++i)
+            pixel[i] = array[pos + i];
+        shouldBe("pixel", "expectedColor");
```
Comment (kenrussell) [inline sdk/tests/conformance2/reading/read-pixels-pack-parameters.html]: Can we make the rows different colors? That'll make the test more meaningful.

**Comment by kenrussell** [comment]
Thanks for the update. One question.

File: sdk/tests/conformance2/reading/read-pixels-pack-parameters.html (hunk 1 above)
Comment (zhenyao) [inline sdk/tests/conformance2/reading/read-pixels-pack-parameters.html]: Sure.  Currently we use clear() to set the fbo color, we just need to switch to shaders and draw()

**Comment by zhenyao** [comment]
Closing this one.

**Pull request closed without merging**